apps/web-staking/src/config/index.tsx

```diff
@@ -1,33 +1,21 @@
-import { defaultWagmiConfig } from '@web3modal/wagmi/react/config'
+import { WagmiAdapter } from '@reown/appkit-adapter-wagmi'
+import { cookieStorage, createStorage } from '@wagmi/core'
 
 // import { cookieStorage, createStorage } from 'wagmi'
-import { arbitrum, arbitrumSepolia } from 'wagmi/chains'
+import { arbitrum, arbitrumSepolia } from '@reown/appkit/networks'
 
 // Get projectId at https://cloud.walletconnect.com
-export const projectId = process.env.NEXT_PUBLIC_APP_ENV === "development" ? "7b8edd4521f6f7e5ab27d84d931ddf9b" : "8f5121741edc292ac7e4203b648d61e2";
+export const projectId = "aa9e5ff297549e8d0cc518d085c28699";
```

Comment:
I'm not sure if we need to create/have separate project Id's for dev/production?